torch/quantization/utils.py

```diff
@@ -6,8 +6,7 @@
 
 def get_combined_dict(default_dict, additional_dict):
     d = default_dict.copy()
-    for k, v in additional_dict.items():
-        d[k] = v
+    d.update(additional_dict)
```

Comment:
oh sorry, I probably wrote this a few times in other places as well..if you can find them please refactor them as well.